ide-common/src/main/kotlin/org/digma/intellij/plugin/updates/UpdatesService.kt

```diff
@@ -228,15 +228,6 @@ class UpdatesService(private val project: Project) : Disposable {
         return appInfo.fullVersion
     }
 
-    // when plugin is not installed it will return 0.0.0
-    @NotNull
-    fun getPluginVersion(): String {
```

Comment:
can you please bring this method back, and make the impl of it:
`SemanticVersionUtil.getPluginVersionWithoutBuildNumberAndPreRelease("0.0.0")`

Reply:
why?